internal/pipeline/examples/basicpipeline/.expected/diff.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/.expected/config.yaml b/.expected/config.yaml
+index 04229ba..2df37b4 100644
+--- a/.expected/config.yaml
++++ b/.expected/config.yaml
+@@ -1,2 +1,5 @@
+ exitCode: 0
+ network: false
++metadata:
```

Comment:
I don't see a pipeline definition that can cause these diff to happen.
Did I miss something?

Reply:
There is a *builtin* function for testing. I think that's the reason.

Reply:
Currently fnFilters is hardcoded to add these annotations. After #1388 they should disappear. Probably better to wait until that is merged before merging this one